docs/introduction/sla.md

```diff
@@ -0,0 +1,208 @@
+# Rahti Container Cloud Service Service Level Agreement DRAFT
+
+## General
+
+This Service Level Agreement (hereafter called SLA) is made between the
+customer, Rahti Container Cloud Service and the service provider, CSC - IT
+Center for Science Ltd., to cover the provision and support of the service as
+described hereafter. Amendments, comments and suggestions must be addressed
+using the communication channels defined in section 8.1. The service provider
+retains right to introduce changes to the infrastructure. If the Customer does
+not accept the changes, this service subscription can be terminated.
+
+## Scope and description of the service
+
+This SLA applies to Rahti Container Cloud Service1 (hereafter referred to as
+Rahti or the Rahti service). The Rahti service is a cloud computing service.
+The Rahti service is based on OKD, which is a distribution of Kubernetes. The
+Rahti serviceallows its customers or users to run their own workloads on top of
+it, often implemented as containers.  The Rahti service provides, but is not
+necessarily limited to, these resources: 
+
+ * Container instances, and groups of container instances
+ * Storage services 
+ * Virtual networks for connecting container instances
+ * Load balancing of traffic to user workloads
+ * Features for replication, rolling updates, auto-recovery and auto-scaling of user workloads
+ * Basic default domain name and TLS for hosted web applications (under rahtiapp.fi)
+
+Customers can manage their resources using a web interface accessible through a
+web browser and through a set of APIs which allow programmatic management of
+resources. In order to access and use the service the customer must have a CSC
+user account. The workloads are separated from other customers’ workloads from
+a network, storage and computational view.
+
+This service is provided under User policy and terms of usage for the Rahti
+contariner cloud service hence by agreeing to this document the customer also
+agrees with CSC General Terms of Use and CSC’s Security Policy.
+
+## Service hours & exceptions
+
+The service is designed to run continuously. However, the following exceptions apply:
+
+Scheduled maintenances:
+
+ * Planned service breaks - Announced to the customer at least three weeks in advance
+ * Unplanned service breaks
+
+## Service components & dependencies
+The service covered by this SLA is made up of the following (technical and logical) service components:
```

Comment:
The list below lists service dependencies, but these are labeled as service components. Service components are presumably different from these. Does FitSM have some sort of a clear definition for what constitutes a service dependency and what constitutes a component? I.e. where do we draw the borders of what is part of the service itself and what are external systems and infrastructure that it depends on.

I also noticed that the content below this subheading is currently the same as in the cPouta SLA.

Reply:
I read the FitSM SLA template, and changed this based my reading of the template. https://fitsm.itemo.org/fitsm-templates-samples-and-guides/
Now it only has services at a very high level.